### PR TITLE
Add help icon with hover-over text next to the filter by tags drop do…

### PR DESCRIPTION
**Pull request by radekkaluzik**
…wn on Insights pages

https://issues.redhat.com/browse/RHCLOUD-28539

### CODE REVIEW

**Comment by radekkaluzik** [comment]
RHCLOUD-28539

File: src/components/GlobalFilter/GlobalFilterMenu.tsx [hunk 1]
```diff
@@ -99,6 +101,7 @@ export const GlobalFilterDropdown: React.FunctionComponent<GlobalFilterDropdownP
     () => (!allowed || isDisabled ? Tooltip : ({ children }: { children: any }) => <Fragment>{children}</Fragment>),
     [allowed, isDisabled]
   );
+  // const inputGroupRef2 = useRef(null);
```
Comment (karelhala) [inline src/components/GlobalFilter/GlobalFilterMenu.tsx:104]: Extra comment
```suggestion
```

File: src/components/GlobalFilter/GlobalFilterMenu.tsx [hunk 2]
```diff
@@ -114,43 +117,80 @@ export const GlobalFilterDropdown: React.FunctionComponent<GlobalFilterDropdownP
...
-                isFilterable
-                onFilter={filter.onFilter || (() => undefined)}
-                filterBy={filter.filterBy as string}
-                showMoreTitle={intl.formatMessage(messages.showMore)}
-                onShowMore={() => setIsOpen(true)}
-                showMoreOptions={{
-                  isLoadButton: true,
-                }}
-              />
+              <InputGroup>
```
Comment (Hyperkid123) [inline src/components/GlobalFilter/GlobalFilterMenu.tsx:120]: There is a lot of extra "stuff" and the file is getting big and messy. Can you extract the group to a separate file?

**Comment by Hyperkid123** [comment]
@radekkaluzik can you add a cypress component test? (Not e2e)

File: src/components/GlobalFilter/GlobalFilterMenu.tsx (hunk 1 above)
Comment (radekkaluzik) [reply src/components/GlobalFilter/GlobalFilterMenu.tsx:104]: Fixed

File: src/components/GlobalFilter/GlobalFilterMenu.tsx (hunk 2 above)
Comment (radekkaluzik) [reply src/components/GlobalFilter/GlobalFilterMenu.tsx]: Moved to a separate component

File: src/components/GlobalFilter/GroupFilterInputGroup.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                onClick: (
+                  e: FormEvent | MouseEventHandler<HTMLInputElement> | undefined,
+                  selected: any,
+                  group: number | undefined,
+                  currItem: boolean | undefined,
+                  groupName: string | undefined,
+                  itemName: string
+                ) => {
+                  generateGlobalFilterEvent((selected?.[groupName as string]?.[itemName as string] as Group)?.isSelected as boolean, item.value);
+                  // item.onClick?.(e, selected, group, currItem, groupName, itemName);
```
Comment (karelhala) [inline src/components/GlobalFilter/GroupFilterInputGroup.tsx:51]: Do we want to disable the `onClick` event?
```suggestion
                  item.onClick?.(e, selected, group, currItem, groupName, itemName);
```

**Review by karelhala — COMMENTED** [review]
Looking good, just one minor question.

File: src/components/GlobalFilter/GroupFilterInputGroup.tsx (hunk 3 above)
Comment (radekkaluzik) [reply src/components/GlobalFilter/GroupFilterInputGroup.tsx:51]: This was all due to a hotjar and besides the condition was always false. Deleted all the unnecessary code.

**Review by karelhala — APPROVED** [review]

**Pull request merged**